egs/gale_arabic/s5b/RESULTS

```diff
@@ -65,6 +65,9 @@ Combined Results for Reports and Conversational WER:
 %WER 32.36 [ 22542 / 69668, 2156 ins, 4184 del, 16202 sub ] exp/tri2b_mmi/decode_it4/wer_11
 %WER 32.50 [ 22640 / 69668, 2393 ins, 3956 del, 16291 sub ] exp/tri2b_mmi/decode_it3/wer_11
 %WER 32.79 [ 22847 / 69668, 2407 ins, 4760 del, 15680 sub ] exp/tri2b_mpe/decode_it3/wer_13
+# WER with train_sat_basis
```

Comment:
sorry I forgot to point it out, @aarora8 would you please remove all old NN results above? They are not replicable any more. thanks.

Reply:
Ok, thank you, I am removing it in a new PR.